lms/djangoapps/branding_by_referer/branding_by_referer.py

```diff
@@ -60,6 +61,10 @@ def process_response(self, request, response):
                 secure=settings.SESSION_COOKIE_SECURE or None
             )
             self.pending_cookie = None
+
+        if getattr(request.resolver_match, 'url_name', None) == 'logout':
```

Comment:
Does this get along with the scenarios where the [LogoutView](https://github.com/eduNEXT/edx-platform/blob/7bdea5d780fd0e667e338bc39e52330df9ea4a65/common/djangoapps/student/views/login.py#L728) has to render so that we logout of services that were authenticated using Oauth.

Reply:
mmm voy a chequear, si no toca checar cuando logout sea el referer